tests/ice_frost.rs

```diff
@@ -272,3 +280,134 @@ fn signing_and_verification_3_out_of_5() {
     assert!(verification_result1.is_ok());
     assert!(verification_result2.is_ok());
 }
+
+#[test]
+fn resharing_from_non_frost_key() {
+    type SchnorrSecretKey = <<Secp256k1Sha256 as CipherSuite>::G as Group>::ScalarField;
+    type SchnorrPublicKey = <Secp256k1Sha256 as CipherSuite>::G;
+
+    // A single party outside of any ICE-FROST scenario, who owns a keypair to perform
+    // Schnorr signatures.
+    let mut rng = OsRng;
+    let single_party_sk: SchnorrSecretKey = SchnorrSecretKey::rand(&mut rng);
+    let single_party_pk: SchnorrPublicKey =
+        <<Secp256k1Sha256 as CipherSuite>::G>::generator() * single_party_sk;
+
+    // Converts this party's keys into ICE-FROST format, simulating a 1-out-of-1 setup.
+    let simulated_parameters = ThresholdParameters::new(1, 1);
+    let frost_sk = IndividualSigningKey::from_single_key(single_party_sk);
+    let frost_pk = GroupVerifyingKey::new(single_party_pk);
+
+    // Start a resharing phase from this single party to a set of new participants.
+    const N_PARAM: u32 = 5;
```

Comment:
```suggestion
    const N_SIGNERS: u32 = 5;
```
Not strong about this, but since the value has a name I think `N_SIGNERS` is more meaningful?

Reply:
I've reused the ones used in benchmarks. I'm not a fan of qualifying them as signers, as they can be dealers whereas the library makes a distinction here (dealers can sign, sole signers cannot redistribute).

Reply:
Alternatively I can just use `N_PARTICIPANTS`, and `THRESHOLD` as you suggested below?

Reply:
I like `N_PARTICIPANTS`!